### PR TITLE
Remove sockets

### PR DESCRIPTION
**Pull request by switters**
Remove all dependence on sockets.

### CODE REVIEW

**Comment by switters** [comment]
@CMClay @slajax - review?

**Comment by CMClay** [comment]
@switters Need to bump the version.

**Comment by CMClay** [comment]
@switters In case you haven't seen it: https://github.com/Retsly/import/wiki/Release-guide

**Comment by CMClay** [comment]
:+1: I guess.

File: index.js [hunk 1]
```diff
@@ -446,16 +371,16 @@ var getDomain = Retsly.prototype.getDomain = function () {
...
 var getOrigin = Retsly.prototype.getOrigin = function () {
-  return document.location.protocol
-      + '//'
-      + document.domain
-      + (document.location.port ? (':' + document.location.port) : '');
+  return document.location.protocol +
+      '//' +
+      document.domain +
+      (document.location.port ? (':' + document.location.port) : '');
 };
```
Comment (slajax) [inline index.js]: I would revert this, it's more readable with `+` first.

**Comment by switters** [comment]
In-office :+1: from @slajax and we're up :rocket:

**Pull request merged**